src/components/TextArea/AutoFocusTextArea.js

```diff
@@ -55,7 +56,7 @@ const styles = StyleSheet.create({
     borderRadius: 8,
     fontFamily: fonts.inter[500],
     fontSize: 14,
-    color: COLORS.black,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color in the input field has been changed from `COLORS.black` to `COLORS.white2`. Make sure that `COLORS.white2` is defined and provides the desired color contrast against the new background color `COLORS.gray100` for readability and accessibility purposes.
```diff
-    color: COLORS.white2,
+    color: COLORS.white2 || COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->